core/src/main/resources/jenkins/model/BuiltInNodeMigration/message.properties

```diff
@@ -20,4 +20,10 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-master=master
+blurb = The term "master" for the main process of Jenkins and built-in node is being retired. \
+  The main process is now called "controller" and the built-in node is called just "built-in node". \
+  The latter change also changes the name-derived label assigned to that node (<code>master</code> to <code>built-in</code>) and the <code>NODE_NAME</code> \
+  environment variable passed to builds running on that node. \
+  Depending on your configuration, this is a potentially breaking change, so it is not applied automatically. \
+  Review label assignments in job configurations and tool installers, and use of the <code>NODE_NAME</code> environment variable in build scripts before applying this change. \
```

Comment:
Should we advise to take a backup first as well?

Reply:
System property override allows using `master` even after migration is done.